test/internals/reply.test.js

```diff
@@ -454,7 +454,76 @@ test('buffer without content type should send a application/octet-stream and raw
     })
   })
 })
+test('Uint8Array without content type should send a application/octet-stream and raw buffer', t => {
+  t.plan(4)
+
+  const fastify = require('../..')()
+
+  fastify.get('/', function (req, reply) {
+    reply.send(new Uint8Array(1024).fill(0xff))
+  })
+
+  fastify.listen({ port: 0 }, err => {
+    t.error(err)
+    t.teardown(fastify.close.bind(fastify))
+
+    sget({
```

Comment:
Can you use `fastify.inject` instead?

Reply:
At the beginning the code used undici, while other tests here used `sget`